site/content/user-guide/http-adapter.md

```diff
@@ -242,7 +242,6 @@ There is a subtle difference between the *device identifier* (*device-id*) and t
 * Request Headers:
```

Comment:
As with the device registry (where it is now fixed), I raise the question here again:
I find it strange that the usage of the HTTP adapter is documented in `Setup Http Adapter`.
And would propose to separate it into two pages instead.

As first time reader I would not expect the very valuable usage documentation of the http adapter in a `setup` page.

Reply:
I tend to agree. I also do not like the `Setup Http Adapter` name so I propose to split it up into `Http Adapter Configuration` and `Http Adapter Usage`. WDYT?

Reply:
:+1:

Reply:
I would like to make this split for all our documents in one go after this PR based on our discussion on IRC